department-of-veterans-affairs/caseflow
# Clarify wording on cancel appeal modal

Resolves #13925

### Description
Updates copy on cancel appeal modal and adds an icon for extra "HEY!"

Also adds an icon to our modal component, updates the style guide, and pulls out some intake copy to COPY.

### Acceptance Criteria
- [x] Cancel appeal modal matches design

### Testing Plan
1. Log in as bvaaabshire and find an ama case assigned to you
2. click "Correct issues" in case details
3. Remove all the issues on the appeal
4. Click "Save" and confirm modal looks like provided copy

### User Facing Changes
BEFORE|AFTER
---|---
![Screen Shot 2020-04-21 at 7 47 13 PM](https://user-images.githubusercontent.com/45575454/79926718-a2429280-840b-11ea-8e39-be0f1cbad59b.png)|![Screen Shot 2020-04-21 at 8 19 22 PM](https://user-images.githubusercontent.com/45575454/79927334-69a3b880-840d-11ea-922a-f39e1c300a84.png)
